packages/manager/apps/hpc-vmware-managed-vcd/src/pages/terminate/TerminateOrganization.page.tsx

```diff
@@ -0,0 +1,73 @@
+import { useContext } from 'react';
+import { useDeleteService } from '@ovh-ux/manager-module-common-api';
+import { DeleteModal } from '@ovh-ux/manager-react-components';
+import {
+  ButtonType,
+  PageLocation,
+  PageType,
+  ShellContext,
+  useOvhTracking,
+} from '@ovh-ux/manager-react-shell-client';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+import { useMessageContext } from '@/context/Message.context';
+
+export default function TerminateOrganization() {
+  const navigate = useNavigate();
+  const { id } = useParams();
+  const { trackClick, trackPage } = useOvhTracking();
+  const { t } = useTranslation('terminate');
+  const { ovhSubsidiary } = useContext(ShellContext).environment.getUser();
+  const { addSuccess } = useMessageContext();
+
+  const onSuccess = () => {
+    trackPage({
+      pageType: PageType.bannerSuccess,
+      pageName: 'delete_managed-vcd_success',
+    });
+    addSuccess({
+      content:
+        ovhSubsidiary === 'US'
+          ? t('terminate_managed_vcd_ftc_success')
+          : t('terminate_managed_vcd_success'),
+      includedSubRoutes: [id],
+    });
+    navigate('..');
+  };
+  const onError = () => {
+    trackPage({
+      pageType: PageType.bannerError,
+      pageName: 'delete_managed_vcd_error',
+    });
+  };
+  const { terminateService, isPending, error, isError } = useDeleteService({
+    onSuccess,
+    onError,
+  });
+
+  return (
+    <DeleteModal
+      isOpen
+      closeModal={() => {
+        trackClick({
+          location: PageLocation.popup,
+          buttonType: ButtonType.button,
+          actionType: 'exit',
+          actions: ['delete_managed-vcd', 'cancel'],
+        });
+        navigate('..');
```

Comment:
```suggestion
        closeModal();
```

Not better ? Just suggestion as we do that in others modals and I think it's more readable

Reply:
:+1:  I'll fix it in the next pull request